### ISSUE TITLE
`sotez.activate` throws an error

### ISSUE THREAD

**Issue by oleiba** [AndrewKishino/sotez#45]
Following code:
```
import Sotez, { Crypto, Utility } from 'sotez';

const sotez = new Sotez('http://alphanet-node.tzscan.io');

(async () => {
    // tz1McQv9msjnEzWrByZB8MvqN1KRzEzqKvFD.json
    const accountJSON = {
        "mnemonic": [
        "neglect",
        "weather",
        "swim",
        "chase",
        "cool",
        "live",
        "song",
        "valid",
        "blouse",
        "crucial",
        "absent",
        "assume",
        "coast",
        "ordinary",
        "culture"
    ],
        "secret": "475f2e4cb0fc3bf82241a17e4795832b8e5a20bc",
        "amount": "22218911835",
        "pkh": "tz1McQv9msjnEzWrByZB8MvqN1KRzEzqKvFD",
        "password": "6FOf8dnJvs",
        "email": "qnjzqtqj.ovpisyao@tezos.example.org"
    };

    const keys = await Crypto.generateKeys(accountJSON.mnemonic.join(" "), accountJSON.email + accountJSON.password);

    await sotez.importKey(keys.sk, keys.passphrase, accountJSON.email);

    const activatedOperation = await sotez.activate(accountJSON.pkh, accountJSON.secret);
    const res = await sotez.awaitOperation(activatedOperation.hash);
})();
```

exits with an error:
```
UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'next_protocol' of undefined
```

This is due to the fact that in `src/sotez.ts` under `prepareOperation` method, the number of promises is estimated incorrectly (assumed to be 4, but when the operation kind is `activate_account` there are only 2).

I am using version 0.5.1.

**Comment by AndrewKishino** [comment]
@oleiba I have pushed a fix and published a new version 0.5.2. Thanks for the issue.

**Comment by oleiba** [comment]
Thanks!
